src/server/server.zig

```diff
@@ -338,29 +341,27 @@ fn update() void { // MARK: update()
 	}
 
 	// Send the entity data:
-	const data = main.stackAllocator.alloc(u8, (4 + 24 + 12 + 24)*userList.len);
-	defer main.stackAllocator.free(data);
+	var writer = BinaryWriter.initCapacity(main.stackAllocator, .big, (4 + 24 + 12 + 24)*userList.len);
```

Comment:
Since this is used on the network, use network.networkEndian.

Reply:
Should we just remove this parameter and make everything big endian automatically?
(That would prevent us from switching in the future, but it's honestly too late for that anyways)

Reply:
Yes, since this is internal tool and we need to consistently use big endian we should avoid giving false choice here. If we ever have a reason to switch it will be better too, as you can just toggle everything at once.

Reply:
> as you can just toggle everything at once.

Well I couldn't because backwards compatibility of save files, but yeah, either way it would be painful.

Reply:
I extracted it to an issue #1222
Until it's implemented please still use network.networkEndian.